airbyte-integrations/connectors/source-postgres/build.gradle

```diff
@@ -6,6 +6,12 @@ plugins {
     id "org.jsonschema2pojo" version "1.2.1"
 }
 
+java {
+    compileJava {
+        options.compilerArgs += "-Xlint:-try,-rawtypes,-unchecked"
```

Comment:
Do we want to file an issue to try to remove this special casing by getting the linter to pass?

Reply:
Same comment applies to the CDK. This isn't a rethorical question.

Reply:
I was going to do it without a new issue, but I can do that

Reply:
Whichever works best for you!